examples/complete/versions.tf

```diff
@@ -4,7 +4,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = ">= 4.20.0"
+      version = ">= 4.0.0, <= 5.15.0"
```

Comment:
why do you have 4 and 5? I would imagine that if you set to one of them is enough

Reply:
This is confining the module to use aws provider version not later than 5.15.0 because of the s3 errors arising from aws provider version 5.17.0. I have tracked to unlock it in the backlog when aws provider version changes